recipes/scalene/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set package_name = "scalene" %}
+{% set version = "1.5.10" %}
+
+package:
+  name: {{ package_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ package_name[0] }}/{{ package_name }}/{{ package_name }}-{{ version }}.tar.gz
+  # See https://pypi.org/project/scalene/#files
+  sha256: 8e2e2ae14acdca56d5bbf33e158814702410f0a72d4e690fce837b811ffadbfc
+
+build:
+  skip: true  # [py<38]
+  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - python
+    - pip
+    - setuptools_scm
+    - wheel
+  run:
+    - python
+    - rich
+    - cloudpickle
+    - numpy
+    - pynvml
```

```suggestion
    - rich >=9.2.0
    - cloudpickle >=1.5.0
    - numpy
    - pynvml >=11.0.0
```
numpy is not listed in setup.py as a requirement and is only imported in the "old" module. Is it really required?

---

It was required for the test, which I thought was necessary, but I guess not?

---

Scratch that, it should in theory be coming from the tarball, but that test _does_ require numpy. I could use a different test that does not.

---

Let me see how that goes.

---

If numpy is only requires for the tests, then it should be listed in the `test/requires` not `requirements/run`.

---

Thanks for the clarification. If this test works, it'll be fine (it's really just to make sure that it produces a profile without a hitch; the numpy test is not needed - it's just what we use in our CI).